src/middlewared/middlewared/etc_files/local/openvpn/client/openvpn_client.conf

```diff
@@ -0,0 +1,43 @@
+<%
+	try:
+		middleware.call_sync('openvpn.client.config_valid')
+	except Exception as e:
+		raise FileShouldNotExist()
+
+	config = middleware.call_sync('openvpn.client.config')
+	root_ca = middleware.call_sync('certificateauthority.query', [['id', '=', config['root_ca']]], {'get': True})
+	client_cert = middleware.call_sync('certificate.query', [['id', '=', config['client_certificate']]], {'get': True})
+%>\
+client
+dev ${config['device_type'].lower()}
+#dev-type ${config['device_type'].lower()} -FIXME: This does not work, is it a openvpn freebsd bug ? Please check
```

Comment:
Yes i am already doing it. The benefit of this working is that we can have a tun/tap interface named `openvpn_server`/`openvpn_client` instead of `tun0` etc. But openvpn is not liking it right now, i am investigating what is triggering that behaviour